onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3730,12 +3745,17 @@ def aten_native_channel_shuffle(self: TensorType, groups: int) -> TensorType:
     raise NotImplementedError()
 
 
+@torch_op("aten::native_dropout", trace_only=True)
```

Comment:
Curious why this needs to be trace only? Is it because of the optional value?

Reply:
mistake.